ninchatsdk/src/main/java/com/ninchat/sdk/managers/NinAudioManager.java

```diff
@@ -0,0 +1,397 @@
+package com.ninchat.sdk.managers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.pm.PackageManager;
+import android.media.AudioDeviceInfo;
+import android.media.AudioManager;
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import org.webrtc.ThreadUtils;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class NinAudioManager {
```

Comment:
Should these be named with Ninchat prefix to be consistent with other files in the project?

Reply:
Backend has traditionally used Nin as prefix and frontend has used Ninchat. Legacy iOS code, which requires three-letter capitalized prefix, has used NIN out of necessity.

In the end, I think consistency within a codebase is most important: use the same prefix for similar things (which need prefixing - I don't know if this one needs a prefix).